src/lib/application.py

```diff
@@ -420,3 +456,126 @@ def is_analysis_complete(self):
+        try:
+            self.wait_find_element(locator_type="image", locator="confirm_exit.png", timeout=5.0)
+            self.click_element(locator_type="image", locator="exit_button.png")
+        except Exception:
+            logging.info("No exit confirmation dialog found !")
+
+    def is_open_mta_perspective(self):
+        """
+        Checks if MTA perspective is already opened in IntelliJ
+
+        Returns:
+            (bool): True or False
+        """
+        try:
+            self.wait_find_element(locator_type="image", locator="mta_perspective_active.png")
+            return True
+        except Exception as exc:
+            logging.debug(
+                "An error occured while finding \
+                MTA perspective tab ! {}".format(
+                    str(exc),
+                ),
+            )
+            if "No matches found" in str(exc):
+                return False
+            else:
+                raise Exception(exc)
+
+    def open_mta_perspective(self):
+        """
+        Opens MTA perspective in IntelliJ
+        """
+        if self.is_open_mta_perspective():
+            logging.info("MTA perspective is already opened !")
+            return
+        else:
+            # Click on the MTA tab in left sidebar
+            self.click_element(locator_type="image", locator="mta_tab.png")
+
+    def run_simple_analysis(self, project, target, packages=[]):
+        """
+        Runs analysis by adding the project and/or packages passed as argument
+
+        Args:
+            project (str): Full name of project to be analysed
+            target (str): Target technology for migration
+            packages (list): List of packages to be added to analysis
+
+        Returns:
+            None
+
+        Steps:
+            1) Click on MTA Configuration tab
+            2) Right click anywhere and create new configuration
+            3) Type project name in source
+            4) Provide mta cli path, if not present
+            5) Select the target technology
+            6) Right click on config name and run
+            7) Confirm analysis has started
+        """
+        config_create_region = self.two_coordinate_locator(
+            locator_type="point", x_coordinate=110, y_coordinate=370,
+        )
+        self.click(config_create_region)
+        self.click(action="right_click")
+        self.press_keys("down")
+        self.press_keys("enter")
+        self.wait_find_element(locator_type="image", locator="mta_config_page_opened.png")
+        # Region defining the configuration name
+        config_name_region = self.define_region(778, 248, 1125, 284)
+        # Provide mta cli path
+        try:
+            self.click_element(locator_type="image", locator="mta_cli_input.png")
+            self.type_text(config_data["mta_cli_path"])
+        except Exception as exc:
+            logging.debug("MTA cli path is already present ! {}".format(exc))
```

Comment:
The test should not fail if the cli path is already present .

Reply:
Yes, this except block ensures, test does not fail, if cli path is already present